app/vmalert/main.go

```diff
@@ -198,6 +205,17 @@ func getAlertURLGenerator(externalURL *url.URL, externalAlertSource string, vali
 	}, nil
 }
 
+func checkConfigs() {
+	u, _ := url.Parse("https://victoriametrics.com/")
+	notifier.InitTemplateFunc(u)
+	if len(*rulePath) == 0 {
+		logger.Fatalf("No rules for validation. Please specify path to file(s) with alerting and/or recording rules using `-rule` flag")
```

Comment:
it will fail anyway in config.Parse() check afaik. So maybe we don't need this check?

Reply:
no error is returned if config is missing or empty, only warning in logs
```
2020-10-18T09:50:03.719Z	warn app/vmalert/config/config.go:224	no groups found in
```